packages/watchexec/watchexec.pacscript

```diff
@@ -0,0 +1,21 @@
+name="watchexec"
+pkgdesc="Executes commands in response to file modifications"
```

Comment:
This is the upstream description, let me know if I should edit it to "A command line tool that...".